sdks/python/apache_beam/runners/common.py

```diff
@@ -98,13 +97,9 @@ def __init__(self,
       self.is_new_dofn = True
 
       # SideInputs
-      self.side_inputs = [side_input
-                          if isinstance(side_input, sideinputs.SideInputMap)
-                          else {global_window: side_input}
-                          for side_input in side_inputs]
+      self.side_inputs = side_inputs
```

Comment:
This is a little confusing now that there is no processing code here; we only need to save the self.side_inputs property when we're using a NewDoFn; can you add a comment to this effect?